controllers/etcd_controller.go

```diff
@@ -254,6 +258,21 @@ func (r *EtcdReconciler) reconcile(ctx context.Context, etcd *druidv1alpha1.Etcd
 		}, err
 	}
 
+	if etcd.Spec.Backup.Copy != nil {
+		values, err := r.getMapFromEtcd(etcd)
+		if err != nil {
+			return ctrl.Result{
+				Requeue: true,
+			}, err
+		}
+		if err := r.runCopyJob(ctx, etcd, values, logger); err != nil {
```

Comment:
Hmm, I don't see where we update the status of the etcd resource to indicate that the job has finished.

I think that the flow should be the following:
1. Trigger the copy job if it has not been triggered.
2. Check if copy job is successfull or not
3. If it is not successful, and this is due to an error, then return an error (the requeue should happen automatically) and update the status of the ETCD resource as errored.
4. If it is not successfull, but is still running normally, return nil error but use Requeue
5. If the job has finished successfully, return nil and update the status of the etcd resource to Successful.